about.md

```diff
@@ -10,6 +10,36 @@ to serve on the ESDS Leadership Committee and/or Working Groups as
 described in our [governance document](https://docs.google.com/document/d/1xjNzJqZjeRzUg5BMipQVUsy2xA1SRlEiyJLpPmUAs9E/edit?usp=sharing).
 
 
+## ESDS Committees and Working Groups
+
+### Leadership Committee
+* Dan Adriaansen, RAL
+* Drew Camron, Unidata
+* John Clyne, CISL
+* Katie Dagon, CGD (stepping down, ~Feb 2024)
+* Orhan Eroglu, CISL
+* Teagan King, CGD
+* Falko Judt, MMM
+* Ryan May, Unidata
+* Mike Levy, CGD
+* Ulrike Romatschk, EOL
+
+### Community Working Group
+* Julia Kent, CISL
+* Katelyn FitzGerald, CISL
```

Comment:
Similar alphabetical nit-picking between Kent and FitzGerald :)

Reply:
Fixed. Thanks